src/elements/content-sidebar/versions/flowTypes.js

```diff
@@ -4,3 +4,4 @@ import type { BoxItemVersion } from '../../../common/types/core';
 
 export type VersionActionCallback = (versionId: string) => void;
 export type VersionChangeCallback = (version: ?BoxItemVersion, additionalVersionInfo: ?AdditionalVersionInfo) => void;
+export type onLoadCallback = (component: string, feature: string, data?: Object) => void;
```

Comment:
How about `SidebarLoadCallback` to be consistent with the types above?

Reply:
Ack!